### PR TITLE
fix pipe-shell initialization order

### PR DESCRIPTION
**Pull request by mmandlis**
ingestion arc can only be initialized after window.ShellApi is set

### CODE REVIEW

File: shells/pipes-shell/node/node.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ import {version, paths, storage, test} from './config.js';
 
 // dependencies
 import {RamSlotComposer} from '../../lib/components/ram-slot-composer.js';
-import {initPipe} from '../pipe.js';
+import {initPipe, initArcs} from '../pipe.js';
```
Comment (shaper) [inline shells/pipes-shell/node/node.js]: This change was made in `node.js` but don't we want to make it in `web.js`?

We are using pipe-shell in web, not node.

Comment (shaper) [reply shells/pipes-shell/node/node.js]: @yuangu-google to confirm which code path makes sense (and if we're not using one, maybe we should delete/archive it?).

Comment (mmandlis) [reply shells/pipes-shell/node/node.js]: I made the changes in both - node and web. 
I thought they are consistent, and wanted to keep them this way, shouldn't they be?

Comment (shaper) [reply shells/pipes-shell/node/node.js]: Ah, I didn't see the web one at the bottom. `node` isn't being used currently, if it's not much work to keep it in sync that's desirable, sure. But if it turns out to be a hassle ongoing I think it'd be ok to remove it and we could pull it back out of the git archive if needed. Up to you.

Comment (mmandlis) [reply shells/pipes-shell/node/node.js]: in this case it was trivial to keep them in sync, so i did.

**Pull request merged**